src/chapters/git/remote-repository.rst

```diff
@@ -0,0 +1,182 @@
+
+Even without an account, we can *clone* our team's work and save it on our own
+machine. Once we have an account (and permission from our team), we can push
+the commits we make from our machine up to GitHub. Neat!
+
+.. _clone-remote-repo:
+
+To clone a remote repository, the terminal command is:
+
+.. sourcecode:: bash
+
+   git clone repo-url
+
+The ``repo-url`` part of the command is where we fill in the web address of the
+repository we want to clone. 
+
+.. admonition:: Try It!
+
+   Since you already initialized the ``git_practice`` directory as a Git
+   repository, it's important to move out of that folder before cloning a new
+   repo!
+
+   #. In VS Code, use the terminal to return to the ``local_practice``
+      directory. Confirm your location with the ``pwd`` command.
+
+      ::
+
+         $ pwd
+         /Users/username/Desktop/local_practice
+
+   #. Now clone an existing project from GitHub. Copy and paste this command
+      into the terminal:
+
+      ::
+
+         $ git clone https://github.com/LaunchCodeEducation/LCHS-git-clone-example.git
+
+   #. Use the file tree or the ``ls`` command so see that a new folder has been
+      saved in the ``local_practice`` directory. Navigate into that folder and
+      look around.
+
+      ::
+
+         $ cd LCHS-git-clone-example
+         $ ls
+         main.py
+         $ python main.py
+
+      *Mac users*: Remember to use ``python3`` instead of ``python`` in the
+      terminal.
+   
+   You can view the code you cloned by opening ``main.py`` in VS Code. Any
+   branches, changes, and commits you make will be saved to your local version
+   of the repository. The remote version remains unchanged by the actions you
+   take on your device.
+
+Saving to a Remote Repository
+-----------------------------
+
+Once we clone a repository to our machine, any changes we make to the project
+stay on our personal device. To update the remote repo, we need to *push* our
+changes up to GitHub.
+
+In this chapter's :ref:`Project <git-project>`, we will go through a detailed
+process to practice pushing and pulling changes from a GitHub repository. Your
+teacher may also have you work through :ref:`Assignment 5 <communication-log>`,
+which adds some partners to your GitHub experience.
+
+For now, we will just look at a summary of the Git commands needed to update a
+remote repository. The process only adds one new step:
+
+#. ``git status``
+#. ``git add .``
+#. ``git commit -m "Message..."``
+#. ``git push origin branch-name``
+
+Step 4 uses the new command ``git push`` to move our local commits up to
+GitHub. The command changes the online repository. ``origin`` makes sure that
+any new files and code do indeed go to the remote (the *origin* of the
+project). ``branch-name`` identifies the branch that the new commits go to.
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   What is the command for making a commit to a remote repository?
```

Comment:
"making a commit" isn't quite accurate. This would better be stated as "pushing a commit," or "moving a commit" (if you don't want to use the leading term "push").

Reply:
I went with "sending a commit".